randovania/games/fusion/logic_database/Sector 6 NOC.txt

```diff
@@ -552,14 +552,17 @@ Extra - room_id: [13]
               # Core-X Requirements
               Missiles
               Combat (Beginner) or 20 DMG Missiles
+              # A way to dodge
+              Combat (Intermediate) or Have Any Jump Upgrade
           Any of the following:
               # Health Requirements
-              Normal Damage ≥ 500
+              Normal Damage ≥ 315
               # Fight Boss Damageless: https://youtu.be/CfgKTcobo0s
               Combat (Hypermode)
-              Combat (Beginner) and Normal Damage ≥ 400
-              Combat (Intermediate) and Normal Damage ≥ 200
-              Combat (Advanced) and Normal Damage ≥ 100
+              Combat (Beginner) and Normal Damage ≥ 250
+              Combat (Intermediate) and Normal Damage ≥ 180
+              Combat (Advanced) and Normal Damage ≥ 120
+              Combat (Expert) and Normal Damage ≥ 60
```

Comment:
I feel like damageless could just be expert honestly

Reply:
I thought so too, but wanted to stay on the safe side here, because
1. AFAIK there's no consistent way on getting the big varia-mass damageless
2. If you mess up once anywhere during this whole 4 minute fight on min reqs, then you will have to redo the whole thing from wherever your last save was (not necessarily the save right before/after varia core)

Reply:
seems like you should save if you are going to fight this boss at 0 Etanks just saying
I dont care enough to block because of this just seems extra

Reply:
the last comment was less implying "forgot to save" and more "saves could be blocked and backtrack is huge"

Reply:
imo distance from a save station shouldn't factor into how difficult a trick should be, rather just the fact you can die/softlock is a good enough indicator (in general).
as for this connection, I agree with Jeff that this is fine as is